src/sxastarter/src/assets/sass/components/layout-container/_media.scss

```diff
@@ -0,0 +1,42 @@
+// Breakpoints variables
+$breakpoints-sm: 370px;
```

Comment:
- Use map for storing default values, so customers can add new entries to the map, e.g. botstrap values
$grid-breakpoints: (
  xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px
);
- prefix breakpoints so it's clear they belong to a layout container
- move variables out to _variables.scss file so it's possible to modify and review them
- declare variables as [default ](https://sass-lang.com/documentation/variables/#default-values) again to simplify modification

Please also test that user can modify and introduce new breakpoints without changes these style files

Reply:
Updated!